src/main/java/net/dv8tion/jda/core/requests/RestAction.java

```diff
@@ -161,13 +162,31 @@
             LOG.error("RestAction queue returned failure: [{}] {}", t.getClass().getSimpleName(), t.getMessage());
         }
     };
+    protected static boolean passContext = false;
```

Comment:
This will change functionality across all running instances of JDA.
Do we want that? Or do we want to provide per-JDA-instance support?